frontend/src/metabase/dashboard/components/parameters/widgets/DateRelativeWidget.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component, PropTypes } from "react";
+
+import RelativeDatePicker from "metabase/query_builder/filters/pickers/RelativeDatePicker.jsx";
+
+import _ from "underscore";
+
+// HACK: easiest way to get working with RelativeDatePicker
+const FILTER_MAPPINGS = {
+    "today":      ["=", null, ["relative_datetime", "current"]],
+    "yesterday":  ["=", null, ["relative_datetime", -1, "day"]],
+    "past7days":  ["TIME_INTERVAL", null, -7, "day"],
+    "past30days": ["TIME_INTERVAL", null, -30, "day"],
+    "lastweek":   ["TIME_INTERVAL", null, "last", "week"],
+    "lastmonth":  ["TIME_INTERVAL", null, "last", "month"],
+    "lastyear":   ["TIME_INTERVAL", null, "last", "year"],
+    "thisweek":   ["TIME_INTERVAL", null, "current", "week"],
+    "thismonth":  ["TIME_INTERVAL", null, "current", "month"],
+    "thisyear":   ["TIME_INTERVAL", null, "current", "year"],
+};
+
+const FILTER_NAMES = {
+    "today":      "Today",
+    "yesterday":  "Yesterday",
+    "past7days":  "Past 7 Days",
+    "past30days": "Past 30 Days",
+    "lastweek":   "Last Week",
+    "lastmonth":  "Last Month",
+    "lastyear":   "Last Year",
+    "thisweek":   "This Week",
+    "thismonth":  "This Month",
+    "thisyear":   "This Year",
+};
```

Comment:
Would it make sense to instead structure these like:

```javascript
const FILTERS = {
    today: {
        name: "Today",
        mapping: ["=", null, ["relative_datetime", "current"]]
    },
    yesterday: {...}, 
    ...}
```

(A single map of maps instead of two separate maps?)

That way it'd be easier to keep track of the different filters and add new ones if we want (e.g. something like "This week")